src/makers/TMGIndustries/hotspots.ts

```diff
@@ -0,0 +1,52 @@
+import HotspotIcon from './hotspot.svg'
+import { MakerHotspot } from '../hotspotMakerTypes'
+import ANTENNAS from './antennas'
+
+const TMGIndustriesHotspotBLE = {
+  name: 'Example Hotspot',
+  icon: HotspotIcon,
+  onboardType: 'BLE',
+  translations: {
+    en: {
+      diagnostic:
+        '<b><white>Diagnostic support allows TMG Industries Maker to identify issues with your Hotspot in a secure way.</white></b>\n\nTMG Industries Maker will never have access to private keys and will only ever be able to access your Hotspot and not any other devices on your Network.\n\nIf you would like to opt-out of diagnostic support please email <purple><b>support@helium.com</b></purple> from the email used to purchase the Hotspot.',
```

Comment:
please replace the helium support email with yours

Reply:
Kindly also guide about the id in index.ts